apps/web/pages/success.tsx

```diff
@@ -299,15 +299,15 @@ export default function Success(props: SuccessProps) {
                           <div className="font-medium">{t("what")}</div>
                           <div className="col-span-2 mb-6">{eventName}</div>
                           <div className="font-medium">{t("when")}</div>
-                          <div className="col-span-2 mb-6">
+                          {/* <div className="col-span-2 mb-6">
                             {date.format("MMMM DD, YYYY")}
                             <br />
                             {date.format("LT")} - {date.add(props.eventType.length, "m").format("LT")}{" "}
                             <span className="text-bookinglight">
                               ({localStorage.getItem("timeOption.preferredTimeZone") || dayjs.tz.guess()})
                             </span>
-                          </div>
```

Comment:
Why were we doing this in the first place? 🤔

Reply:
Looks like a merge conflict resolution problem to me. Not sure. @leog @joeauyeung  Do you have any idea